students/philip_behrend/lesson05/except_exercise.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+
+"""
+An exercise in playing with Exceptions.
+Make lots of try/except blocks for fun and profit.
+
+Make sure to catch specifically the error you find, rather than all errors.
+"""
+import os
+os.chdir(r'C:\Users\Gemini\UW_PYTHON\SP_Online_PY210\students\philip_behrend\lesson05')
```

Comment:
avoid hardcoding your personal directory structure in assignments please - this will prevent us from running your code.